src/amo/components/HomeHeroBanner/index.js

```diff
@@ -86,21 +91,19 @@ export class HomeHeroBannerBase extends React.Component<Props> {
               backgrounds, local weather & more`)}
         </p>
       </HeroSection>,
-      <HeroSection key="hero-11" linkTo="/addon/share-backported/">
-        <h3>{i18n.gettext('Share Backported')}</h3>
+      <HeroSection key="hero-11" linkTo="/addon/tree-style-tab/">
+        <h3>{i18n.gettext('Tree Style Tab')}</h3>
 
         <p>
-          {i18n.gettext(`Put a social media ‘Share’ button into Firefox
-              toolbar`)}
+          {i18n.gettext(
+            `Have a ton of open tabs? Organize them in a tidy sidebar.`,
```

Comment:
This one's odd, because we have punctuation in the middle of it, but I guess we still need to remove the `.` at the end? Maybe @devaneymoz can confirm.